data/json/mapgen/map_extras/scientists.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "corpse_blood_gibs_science_3x3",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "place_items": [ { "item": "map_extra_science", "x": [ 0, 2 ], "y": [ 0, 2 ], "chance": 100 } ],
+      "place_fields": [ { "field": "fd_blood", "x": [ 0, 2 ], "y": [ 0, 2 ] }, { "field": "fd_gibs_flesh", "x": [ 0, 2 ], "y": [ 0, 2 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_science",
+    "object": {
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "           1            ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": { "1": "t_null" },
+      "monsters": { " ": { "monster": "GROUP_NETHER_CAPTURED", "chance": 1, "density": 0.0001 } },
+      "nested": {
+        " ": { "chunks": [ [ "corpse_blood_gibs_science_3x3", 1 ], [ "null", 150 ] ] },
```

Comment:
I want to double check those chunks are in the game already as I don't see them here and when I searched my local github for `corpse_blood_gibs_science_3x3` my search game back with zero responses.

Reply:
This chunk is located in this very file, at the very top.